src/test/tests/databases/mili.py

```diff
@@ -449,6 +452,81 @@ def TestGlobalIntegrationPoint():
     DeleteAllPlots()
     CloseDatabase(db_path)
 
+def TestMaterialEdgeLines():
+    TestSection("Material edge lines")
+
+    db_path = multi_domain_path + "/d3samp6.plt.mili"
+    OpenDatabase(db_path)
+
+    AddPlot("FilledBoundary", "materials1")
+    DrawPlots()
+
+    AddPlot("FilledBoundary", "materials1")
+
+    silr = SILRestriction()
+    silr.SuspendCorrectnessChecking()
+    silr.TurnOffAll()
+    for silSet in (24,29,34,39,44,49,54,59):
+        silr.TurnOnSet(silSet)
+    silr.EnableCorrectnessChecking()
+    SetPlotSILRestriction(silr ,1)
+
+    SetActivePlots(1)
+    FilledBoundaryAtts = FilledBoundaryAttributes()
+    FilledBoundaryAtts.colorType = FilledBoundaryAtts.ColorBySingleColor  # ColorBySingleColor, ColorByMultipleColors, ColorByColorTable
```

Comment:
Since you are coloring by single color here, is it really necessary to call `SetMultiColor`?
Are there any other atts that don't need to be specifically called out here (eg don't change from default, like `colorTableName` , `invertColorTable`, etc?)  IMHO, it is easier to understand what options are really being set if what is changed is only what is truly necessary.  Same would apply to View3DAtts if you subscribe to my point of view.